scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2696,69 +2696,69 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
   }
 
   private def modifier(isLocal: Boolean): Mod = {
-    val mod = autoPos(token match {
+    val mod = token match {
       case t: Unquote => unquote[Mod](t)
       case t: Ellipsis => ellipsis[Mod](t, 1)
-      case _: KwAbstract => next(); Mod.Abstract()
-      case _: KwFinal => next(); Mod.Final()
-      case _: KwSealed => next(); Mod.Sealed()
-      case _: KwImplicit => next(); Mod.Implicit()
-      case _: KwLazy => next(); Mod.Lazy()
-      case _: KwOverride if !isLocal => next(); Mod.Override()
+      case _: KwAbstract => atCurPosNext(Mod.Abstract())
+      case _: KwFinal => atCurPosNext(Mod.Final())
+      case _: KwSealed => atCurPosNext(Mod.Sealed())
+      case _: KwImplicit => atCurPosNext(Mod.Implicit())
+      case _: KwLazy => atCurPosNext(Mod.Lazy())
+      case _: KwOverride if !isLocal => atCurPosNext(Mod.Override())
       case _: KwPrivate if !isLocal => privateModifier()
       case _: KwProtected if !isLocal => protectedModifier()
       case _ =>
         token.toString match {
-          case soft.KwInline() => next(); Mod.Inline()
-          case soft.KwInfix() => next(); Mod.Infix()
-          case soft.KwOpen() if !isLocal => next(); Mod.Open()
-          case soft.KwOpaque() => next(); Mod.Opaque()
-          case soft.KwTransparent() => next(); Mod.Transparent()
+          case soft.KwInline() => atCurPosNext(Mod.Inline())
+          case soft.KwInfix() => atCurPosNext(Mod.Infix())
+          case soft.KwOpen() if !isLocal => atCurPosNext(Mod.Open())
+          case soft.KwOpaque() => atCurPosNext(Mod.Opaque())
+          case soft.KwTransparent() => atCurPosNext(Mod.Transparent())
           case n =>
             val local = if (isLocal) "local " else ""
             syntaxError(s"${local}modifier expected but $n found", at = token)
         }
-    })
+    }
     newLinesOpt()
     mod
   }
 
   def quasiquoteModifier(): Mod = entrypointModifier()
 
   def entrypointModifier(): Mod = {
-    val mod = autoPos {
-      def annot() = annots(skipNewLines = false) match {
```

Comment:
please take a look; changed `skipNewLines` to true, since we are ok skipping newlines.

Reply:
This should be ok. Adding skipNewLines just make it a bit more permisive. We anyway usually remove newlines between annots.